### DISCUSSION TITLE
`hugo new tutorials/my-tutorial.md` command hangs

### DISCUSSION THREAD

**Discussion by FuadEfendi** [hbstack/.github#12] [Issues and bugs]
I am not sure if it is bug of the framework or not, `hugo new tutorials/my-tutorial.md` command just hangs with no response.

**Comment by FuadEfendi** [comment]
https://github.com/gohugoio/hugo/issues/9166

**Comment by FuadEfendi** [comment]
Ok; now it is different (but still doesn't work):
1. Upgrade: `hugo mod get` 
2. After the upgrade, it is probably responsive (at least I can see some output), but still didn't finish after few minutes:
```
fefendi@FE-MBP-2021 www.fuadefendi.com % hugo new blog/hello/index.md -v
hugo: downloading modules …
hugo: collected modules in 4598 ms
```

Perhaps it is HUGO bug, specific to modules, see related https://github.com/gohugoio/hugo/issues/9166

**Comment by razonyang** [comment]
It's probably not an issue related to the framework.

Closing this per above comments.

**Comment by FuadEfendi** [comment]
For those who has similar issue: run `ps -ef | grep hugo` and ensure you don't have hugo server running. I had it (probably because MacOS automatically started hidden process when upgraded&restarted at night); after killing it it works.
